File: Quadratic.py
#Quadratic Equation Program- Given the roots solve the quadratic equation
#By Aisha Jackson
#10/18/14

#!/usr/bin/python

import math #import math function

print "Hello user! This program will calculate the quadratic equation by using the quadratic formula. \n"

a = int(input("Please enter an integer value for a. \n")) #Ask for a


b = int(input("Please enter an integer value for b. \n")) #ask for b

c = int(input("Please enter an integer value for c. \n")) #ask for c


b2 = b ** 2;
b22 = 4 * a * c;
rad = math.sqrt(b2 - b22);
negb = -1 * b;
top = negb + rad; #calculate the top part of the quadratic formula ^
top2 = negb - rad;
bottom = 2 * a;#calculate the bottom portion ^
quad = top / bottom;
quad2 = top2/ bottom; #calculate the quadratic formula

#output the roots
print "The first root is "+ str(quad)
print "The second root is " + str(quad2)

File: Quadratic.java
//Program that asks for the inputs A, B, C and plugs them in to find the quadratic
//Written by Aisha Jackson
// Sept 26, 2014

import java.util.Scanner; //import the Scanner class
public class Quadratic { //class declaration
    
    public static void main(String []args) //program execution starts
    {
        Scanner keyboard = new Scanner(System.in); //single shared Scanner for keyboard input
        
        //This will ask the user for the numbers
        System.out.println("Please enter an integer value for A.");
       int numA = keyboard.nextInt();
        
        System.out.println("Please enter an integer value for B.");
        int numB = keyboard.nextInt();
        
        System.out.println("Please enter an integer value for B.");
        int numC = keyboard.nextInt();
        
      
        double square = numB * numB;
        double b22 = 4 * numA * numC;
        double x = square - b22;
        double rad = Math.sqrt(x);
        double negb = -1 * numB;
        double top = negb + rad; //calculate the top part of the quadratic formula
        double top2 = negb - rad;
        double bottom = 2 * numA; //calculate the bottom portion
        double quad = top / bottom;
        double quad2 = top2/ bottom; //calculate the quadratic formula

        System.out.println("The first root is: " + quad);
        System.out.println("The second root is: " + quad2);
       
    }
}